control/iosys.py

```diff
@@ -918,7 +918,6 @@ def __init__(self, syslist, connections=[], inplist=[], outlist=[],
             # TODO: look for duplicated system names
             self.syslist_index[sys.name] = system_count
             system_count += 1
```

Comment:
Sorry, my previous suggestion deleted the whole line, not just the whitespace. Please revert.

```suggestion
            system_count += 1

```